iree/compiler/Codegen/Dialect/LoweringConfig.td

```diff
@@ -200,14 +208,6 @@ def IREECodegen_CompilationInfoAttr :
   }];
 
   let builders = [
-    AttrBuilder<(ins "TileSizesListTypeRef":$tileSizes,
```

Comment:
Why are the builders removed?

Reply:
They are not used anywhere, so I deleted them. I just don't want to pass the tile interchange array to it without any testing. I can add them back if you think they are helpful.

Reply:
We need this externally for auto-tuner to generate the attribute. Maybe add a comment here for future reference.